public/app/features/explore/Logs/LogsContainer.tsx

```diff
@@ -59,45 +60,71 @@ interface LogsContainerProps extends PropsFromRedux {
 
 interface LogsContainerState {
   logDetailsFilterAvailable: boolean;
+  /**
+   * LogContext support for mixed data sources.
+   * @alpha
+   */
+  logContextSupport: Record<string, DataSourceApi<DataQuery> & DataSourceWithLogsContextSupport<DataQuery>>;
 }
 
 class LogsContainer extends PureComponent<LogsContainerProps, LogsContainerState> {
   state: LogsContainerState = {
     logDetailsFilterAvailable: false,
+    logContextSupport: {},
   };
 
   componentDidMount() {
-    this.checkFiltersAvailability();
+    this.checkDataSourcesFeatures();
   }
 
   componentDidUpdate(prevProps: LogsContainerProps) {
-    this.checkFiltersAvailability();
+    this.checkDataSourcesFeatures();
   }
 
-  private checkFiltersAvailability() {
+  private checkDataSourcesFeatures() {
     const { logsQueries, datasourceInstance } = this.props;
 
-    if (!logsQueries) {
+    if (!logsQueries || !datasourceInstance) {
       return;
     }
 
-    if (datasourceInstance?.modifyQuery || hasToggleableQueryFiltersSupport(datasourceInstance)) {
-      this.setState({ logDetailsFilterAvailable: true });
+    let newState: LogsContainerState = { ...this.state, logDetailsFilterAvailable: false };
+
+    // Not in mixed mode.
+    if (datasourceInstance.name !== MIXED_DATASOURCE_NAME) {
```

Comment:
I think we should do that with `datasourceInstance.type === 'mixed'`. However I also saw this: https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/mixed/MixedDataSource.ts#L30-L32

@ifrost any particular reason to do it based on the DS name rather than the type?

Reply:
Switched to use `uid` like https://github.com/grafana/grafana/blob/main/public/app/features/explore/state/query.ts#L388C2-L388C2.

Reply:
I think for mixed data source `datasource.type` is `"datasource"` not `"mixed"`. I think in theory you can check `.meta.mixed` or `.meta.id` here. Checking by name is just the most commonly used across the code base. Probably the reason why the name is used so commonly it's just it was more convenient as it is working the same way regardless of using `DataSourceSettings` (pre-loaded for all data sources) or `DataSourceApi` (needs extra async call).

Reply:
Oh, right. I think switching to `uid` makes sense as it makes sure it is unique.